src/NoteMouseHandler/SelectionMouseHandler.js

```diff
@@ -1,72 +1,71 @@
 import _ from "lodash"
-import MouseHandler, { defaultActionFactory, getLocal } from "./NoteMouseHandler"
+import MouseHandler, { defaultActionFactory } from "./NoteMouseHandler"
 import { createNoteAction } from "./PencilMouseHandler"
 import Rect from "../model/Rect"
 import { pointSub } from "../helpers/point"
 
+const defaultCursor = "crosshair"
+
 export default class SelectionMouseHandler extends MouseHandler {
-  constructor(emitter) {
-    super(emitter, [defaultActionFactory, actionFactory], getCursor)
+  constructor() {
+    super([defaultActionFactory, actionFactory], getCursor, defaultCursor)
   }
 }
 
-function actionFactory(e, ctx) {
+function actionFactory(local, ctx, e) {
   if (e.relatedTarget) {
     return null
   }
 
-  const selectionRect = ctx.selection.getBounds(ctx.transform)
-  const cpos = getLocal(e)
-  const clicked = selectionRect && selectionRect.contains(cpos.x, cpos.y)
-  if (!clicked) {
-    if (e.nativeEvent.detail == 2) {
-      return createNoteAction(ctx)
+  const { selection, transform } = ctx
+  if (selection && selection.enabled) {
+    const selectionRect = selection.getBounds(transform)
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->